rlenvs/Env.lua

```diff
@@ -34,7 +38,15 @@ end
 
 function Env:start()
     self.currentStep = 1
-    return self:_start()
+    local obs = self:_start()
+    return obs
+end
+
+function Env:render()
+    if self.qt and self.getDisplay then
+        self.window = self.window == nil and image.display({ image = self:getDisplay(), zoom = 10 }) or self.window
```

Comment:
Shall I set the default to 10 for Catch or leave this exposed for the user as well?